core-primitives/ocall-api/src/lib.rs

```diff
@@ -93,6 +93,7 @@ pub trait EnclaveOnChainOCallApi: Clone + Send + Sync {
 		&self,
 		extrinsics: Vec<OpaqueExtrinsic>,
 		parentchain_id: &ParentchainId,
+		await_each_inclusion: bool,
```

Comment:
necessary to wait for vault to be created before trying to send an extrinsic as vault